Topics/Tech_Stacks/Odata.md

```diff
@@ -4,16 +4,24 @@
 
 ### [Introduction](#introduction-1)
 
+### [What is OData](#what-is-odata-1)
+
 ### [Getting Started with OData](#getting-started-with-odata-1)
 
 ### [OData Basics](#odata-basics-1)
 
+### [How to Use OData for RESTful APIs](#how-to-use-odata-for-restful-apis-1)
```

Comment:
The “How to Use OData for RESTful APIs” table of contents title isn’t linking to the correct section